gnovm/pkg/gnolang/op_eval.go

```diff
@@ -43,19 +43,26 @@ func (m *Machine) doOpEval() {
 		switch x.Kind {
 		case INT:
 			x.Value = strings.ReplaceAll(x.Value, "_", "")
+			isNegative := false
+			if strings.HasPrefix(x.Value, "-") {
+				isNegative = true
+				x.Value = strings.TrimPrefix(x.Value, "-")
+			}
```

Comment:
wait, I don't think this is necessary. I think this is handleded separately as its own unary operator?

Reply:
Oh, now that I take another look, it's running fine without that processing, so maybe I'm looking at it wrong. 😅

I'll fix it on Monday. thanks!